components/brave_wallet/browser/internal/orchard_block_scanner.h

```diff
@@ -26,15 +28,20 @@ class OrchardBlockScanner {
   struct Result {
     Result();
     Result(std::vector<OrchardNote> discovered_notes,
-           std::vector<OrchardNoteSpend> spent_notes);
-    Result(const Result&);
-    Result& operator=(const Result&);
+
+           std::vector<OrchardNoteSpend> spent_notes,
```

Comment:
how many OrchardNoteSpend and OrchardNote should we expect to see here (order of magnitude)? If we are dealing with very large lists then any operations on them should be off the UI thread

Reply:
then please annotate with `base::AssertLongCPUWorkAllowed`

Reply:
Added